### PR TITLE
SIG: add description for k8s SIG

### PR DESCRIPTION
**Pull request by Yisaer**

### CODE REVIEW

File: special-interest-groups/sig-k8s/roles-and-organization-management.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# SIG K8S Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the K8S SIG is to have
```
Comment (tennix) [inline special-interest-groups/sig-k8s/roles-and-organization-management.md]: Do not break the line manually.

File: special-interest-groups/sig-k8s/roles-and-organization-management.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+**How to rejoin**:
+
+* Should meet the requirements for Active Contributor again.
+
+### Reviewer
+
+**Automatically demoted**:
+
+For a Reviewer to be demoted to an Active Contributor,  he/she must satisfy at
+least 1 of the following conditions:
```
Comment (tennix) [inline special-interest-groups/sig-k8s/roles-and-organization-management.md]: ditto

File: special-interest-groups/sig-k8s/roles-and-organization-management.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Haven't appeared in the SIG meeting without any reason for 3 times continuously.
+* At least 2 Committers or Maintainers agree that the Reviewer is not qualified.
+
+**How to retain**:
+
+* Have reviewed or merged 2 PRs within 1 month.
+
+### Voluntarily Quit or Demote
+
+Please email to [Yanqing Zhang](mailto:zhangyanqing@pingcap.com) if you are
```
Comment (tennix) [inline special-interest-groups/sig-k8s/roles-and-organization-management.md]: ditto

File: special-interest-groups/sig-k8s/roles-and-organization-management.md (hunk 1 above)
Comment (Yisaer) [reply special-interest-groups/sig-k8s/roles-and-organization-management.md]: updated

File: special-interest-groups/sig-k8s/member-list.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# SIG K8S Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2020-02-21
+
+## Tech Leads
```
Comment (zz-jason) [inline special-interest-groups/sig-k8s/member-list.md]: there are too many leads, could you limit the number to no more than 3?

File: special-interest-groups/sig-k8s/roles-and-organization-management.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+The minimal requirement for contributors to join the K8S SIG is to have the **Active Contributor** community role in the TiDB-Operator repository.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 4 PRs merged within 2 months.
```
Comment (zz-jason) [inline special-interest-groups/sig-k8s/roles-and-organization-management.md]: In here [TiDB Community Architecture](https://github.com/pingcap/community/tree/master/architecture) we described the general requirements for role promotions. Please follow these principles.

File: special-interest-groups/sig-k8s/member-list.md (hunk 4 above)
Comment (Yisaer) [reply special-interest-groups/sig-k8s/member-list.md]: updated

File: special-interest-groups/sig-k8s/roles-and-organization-management.md (hunk 5 above)
Comment (Yisaer) [reply special-interest-groups/sig-k8s/roles-and-organization-management.md]: updated

File: special-interest-groups/sig-k8s/member-list.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+Leads:
+* [tennix](https://github.com/tennix), [PingCAP](https://pingcap.com/en/)
```
Comment (tennix) [inline special-interest-groups/sig-k8s/member-list.md]: I suggest @cofyc and @Yisaer as the lead and co-lead.

Comment (Yisaer) [reply special-interest-groups/sig-k8s/member-list.md]: updated. PTAL

File: special-interest-groups/sig-k8s/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+See [SIG K8S Member List](./member-list.md)
+
+## Meetings
+
+* TBD
+
+## Contact
+
+* Slack: channel #sig-tidb-operator in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (zhangjinpeng1987) [inline special-interest-groups/sig-k8s/README.md]: I don't think tidb-operator is proper for this sig, maybe tidb-on-cloud?

Comment (Yisaer) [reply special-interest-groups/sig-k8s/README.md]: I agreed. As #sig-tidb-cloud is an existed channel, we should rename it as #sig-tidb-on-cloud

Comment (tennix) [reply special-interest-groups/sig-k8s/README.md]: There is no such channel yet.

**Pull request merged**